### ISSUE TITLE
A new user cannot see a folder shared with "Everyone"

### ISSUE THREAD

**Issue by hirunatan** [icewind1991/group_everyone#19]
Hello. I've just installed the plugin in my Nexctloud instnce. I have shared a folder with the Everyone group, and all existing people sees it.

But I have added a new Nextcloud user, and he doesn't see the folder, even though he seems to belong to the group Everyone, as all other people.

Is this a bug? Or do I need to do something more? Thanks.

**Comment by coach1988** [comment]
@hirunatan Probably related to https://github.com/nextcloud/server/issues/25062

**Comment by nursoda** [comment]
It seems that the alternative app [Auto Groups](https://apps.nextcloud.com/apps/auto_groups) solves this if you enable the Login Hook. But I am irritated that this app even is prone to this glitch as it should create a virtual group that consists of all users.

**Comment by hirunatan** [comment]
Yes, i'm currently using this workaround. It's just I find it cleaner to allow sharing with everyone, than having a group to which everyone belongs.
